packages/components/filters/src/filter-menu/filter-menu.tsx

```diff
@@ -1,20 +1,106 @@
-import DropdownMenu from '@commercetools-uikit/dropdown-menu';
+import { type ReactNode } from 'react';
+import { css } from '@emotion/react';
+import Constraints from '@commercetools-uikit/constraints';
+import { designTokens } from '@commercetools-uikit/design-system';
+import * as Popover from '@radix-ui/react-popover';
 import { Footer } from './footer';
 import { Header } from './header';
 import { TriggerButton } from './trigger-button';
 
+export type TAppliedFilterValue = {
+  value: string;
+  label: ReactNode;
+};
+
 export type TFilterMenuProps = {
   /**
-   * This is a stub prop!
+   * unique identifier for the filter
+   */
+  filterKey: string;
+  /**
+   * formatted message to display the filter's name
    */
-  label: string;
+  label: ReactNode;
+  /**
+   * the input in which the user selects values for the filter
+   */
+  renderMenuBody: () => ReactNode;
+  /**
+   * the input in which the user can select which operator should be used for this filter
+   */
+  renderOperatorsInput?: () => ReactNode;
```

Comment:
Is this related to the `renderOperandsInput` operator described in the filter header ticket? They seem to be serving the same purpose but want to be sure if they are in different context. 
Asking to know if I need to align the name with this prop.

Reply:
yeah, sorry, it should be `renderOperatorsInput` all the way down

Reply:
Got it,  updated my PR. Thanks!